codalab/bundles/run_bundle.py

```diff
@@ -21,65 +21,301 @@
 class RunBundle(DerivedBundle):
     BUNDLE_TYPE = 'run'
     METADATA_SPECS = list(DerivedBundle.METADATA_SPECS)  # type: List
-    # Note that these are strings, which need to be parsed
-    # Request a machine with this much resources and don't let run exceed these resources
-    # Don't format metadata specs
-    # fmt: off
-    METADATA_SPECS.append(MetadataSpec('request_docker_image', str, 'Which docker image (either tag or digest, e.g., codalab/default-cpu:latest) '
```

Comment:
I personally think that the original formatting is easier to read and see all the fields at one glance...what do you think?

Reply:
I generally agree, but things like this hurts my heart:

![Screen Shot 2022-09-14 at 10 24 44 PM](https://user-images.githubusercontent.com/25855750/190321456-0666f4c6-c9fc-4c23-a2d9-4a2ee2800d8a.png)

Reply:
One option I've been vaguely thinking about is storing the definitions somewhere else. But I don't want to do that in the scope of this PR

Reply:
I think putting the definitions here is nice so that everything is in one place.  I agree that long lines and weird wrapping is not-ideal either...  I only have a weak preference for the single lines, so feel free to wrap if you feel strongly about it.